src/Symfony/Component/Validator/Mapping/Loader/XmlFileLoader.php

```diff
@@ -30,9 +30,8 @@ class XmlFileLoader extends FileLoader
      */
     protected array $classes;
 
-    public function __construct(string $file)
+    public function __construct(protected string $file)
```

Comment:
I'm not sure we want to use CPP in these classes, given they use a property from a parent class.

Redefining the property in a child class so we can use CPP doesn't seem to add much, while it does mean we duplicate the property definition multiple times (making it harder to change anything there).